src/Kirel.Identity.Server.API/appsettings.Development.json

```diff
@@ -19,6 +22,39 @@
     "Admin": {
       "Reset": false,
       "Password": "Admin@123"
-    }
+    },
```

Comment:
I think we should seperate maintenance and data seeding, it's a different things. Maintenance is used always, each app start, but data seeding, only first app run, can you seperate this methods and configs?